main.go

```diff
@@ -842,23 +842,27 @@ nodes.
 }
 
 var runCmd = &cobra.Command{
-	Use:   "run <cluster> <command> [args]",
-	Short: "run a command on the nodes in a cluster",
+	Use:     "run <cluster> <command> [args]",
+	Aliases: []string{"ssh"},
+	Short:   "run a command on the nodes in a cluster",
 	Long: `Run a command on the nodes in a cluster.
 `,
-	Args: cobra.MinimumNArgs(2),
+	Args: cobra.MinimumNArgs(1),
 	Run: wrap(func(_ *cobra.Command, args []string) error {
 		c, err := newCluster(args[0], false /* reserveLoadGen */)
 		if err != nil {
 			return err
 		}
 
+		if len(args) == 1 || len(c.Nodes) == 1 {
```

Comment:
I think this condition should be `if len(args) == 1`. If there's a non-interactive command, it should always go through `c.Run`, instead of switching based on whether it targets one or multiple nodes.